sympy/physics/control/lti.py

```diff
@@ -4302,3 +4302,107 @@ def is_controllable(self):
 
         """
         return self.controllability_matrix().rank() == self.num_states
+
+
+class StateSpaceSeries(StateSpace):
```

Comment:
Hi, this is surely something we can add. 
But this feels strange. As we already have a classes like this -
https://github.com/sympy/sympy/blob/d06f908005001356af8c722507882b218c198602/sympy/physics/control/lti.py#L1326-L1329

If this handles transfer functions that are SISOLTI in nature, shouldn't this be directly expanded as StateSpaceModels and are nothing but systems (just like transfer functions).

We also have the MIMOSeries class -
https://github.com/sympy/sympy/blob/d06f908005001356af8c722507882b218c198602/sympy/physics/control/lti.py#L1593-L1596

We also identify whether the StateSpaceModel is SISO or MIMO when it is created.
https://github.com/sympy/sympy/blob/d06f908005001356af8c722507882b218c198602/sympy/physics/control/lti.py#L3678-L3683 

So here I can see how the models can be divided as well.

Reply:
Let me know your thoughts on - 
Expanding Series to hold all SISO systems (`TransferFunction`, SISO `StateSpace`) and MIMOSeries to hold all MIMO systems (`TransferFunctionMatrix`, MIMO `StateSpace`).

Reply:
Yes you are right, we can expand the existing classes to work for `StateSpace` , I was just experimenting to know your thoughts on this kind of implementation.

Reply:
Transfer functions and state spaces with matching numbers of input-output should be able to be combined. I may suggest working through the existing classes and clean up some possible blockers for this, rather than working around by introducing new classes.

Reply:
I think that for `StateSpace` connections , we don't need to change into `TransferFunction` after doing it's `.doit()`. In this case the output after `doit()` should be `StateSpace` rather than `TransferFunction`.

Let's say I've a `StateSpace` object `ss1` and `TransferFunction` object `tf1` if they will be connected in `Series` as `ser = Series(ss1, tf1)`. Then what should `ser.doit()` should return ?

 If it will return a equivalent `StateSpace` object then we have to convert the `tf1` to `StateSpace` and then multiply it with `ss1` to get the output. Do you agree with this point ?

I've a local copy of code which works with current `Series` and `MIMOSeries` but only for `StateSpace` systems. I'll try to make it work with `TransferFunction` also.